### ISSUE TITLE
Talkback does not announce dates correctly for English United Kingdom dd/mm/yy format

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#347]
```
What steps will reproduce the problem?
1. Enable Talkback, Explore By Touch. Sound Feedback off.
2. Open Android Settings
3. Open About Phone.
4. Open Status.
5. Explore by touch each item.

What is the expected output? What do you see instead?
Talkback should read each item once.
But Talkback costantly re-reads each item.
Have to use the Power button to escape out of it.
Back and Home buttons don't work.

What version of the product are you using? On what operating system?
Talkback 3.5.1
Android 4.3 and 4.4.2

Please provide any additional information below.
Tested on a Motorola Moto G



```

Original issue reported on code.google.com by `mossieg...@gmail.com` on 31 Jan 2014 at 9:18

**Comment by GoogleCodeExporter** [comment]
```
Please close this issue. The summary is wrong.
If you could see the problem I have using this web page you wouldn't be 
surprised I screwed up...
```

Original comment by `mossieg...@gmail.com` on 31 Jan 2014 at 9:23
